XmlFormatter/src/Windows/HotfolderEditor.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using XmlFormatter.src.Enums;
+using XmlFormatter.src.Hotfolder;
+using XmlFormatter.src.Interfaces.Formatter;
+using XmlFormatter.src.Interfaces.Hotfolder;
+
+namespace XmlFormatter.src.Windows
+{
+    public partial class HotfolderEditor : Form
+    {
+        private bool editMode;
+
+        private bool saved;
+        public bool Saved => saved;
+
+        private Dictionary<string, Type> formatters;
+
+        private IHotfolder hotfolder;
+        public IHotfolder Hotfolder => hotfolder;
+
+        public HotfolderEditor(IHotfolder hotfolder)
+        {
+            editMode = hotfolder == null ? false : true;
+            formatters = new Dictionary<string, Type>();
+           
+            InitializeComponent();
+
+            CB_Formatter.Enabled = !editMode;
+            this.hotfolder = hotfolder;
+        }
+
+
+        private void HotfolderEditor_Load(object sender, EventArgs e)
+        {
+            Type type = typeof(IFormatter);
+            Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'assemblies'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053278)